pegasus/sites.v3/code.org/public/css/hoc-banner.scss

```diff
@@ -1,9 +1,5 @@
 @import 'color.scss', 'font.scss', 'breakpoints.scss';
 
-#homepage #hero {
-  overflow: hidden;
-}
-
```

Comment:
Removing this for now. We're planning on updating the homepage this year so this code should be irrelevant by the next HoC.